src/odemis/gui/plugin/__init__.py

```diff
@@ -494,6 +495,29 @@ def showProgress(self, future):
         # press the "cancel" button, if such button exists, otherwise provide
         # such a button. That button will call cancel() on the future.
 
+    @call_in_wx_main
+    def add_acquisition_info(self, txt=None, lvl=logging.INFO):
+        """
+        Displays acquisition info above progress bar.
+        input
+            txt (String): text to be displayed
```

Comment:
`str or None`. Describe what happens when passing None.

Also, no need to put an indentation for the parameters description.